src/Mono.Android/Mono.Android.targets

```diff
@@ -260,4 +268,37 @@
       <FileWrites Include="$(IntermediateOutputPath)CheckApiCompatibility.stamp" />
     </ItemGroup>
   </Target>
+
+  <Target Name="UpdateExternalDocumentation">
+    <MSBuild Projects="$(MSBuildThisFileDirectory)Mono.Android.csproj"
+        Properties="TargetFramework=monoandroid10"
+        Targets="_UpdateExternalDocumentation"
+    />
+  </Target>
+  <Target Name="_UpdateExternalDocumentation">
+    <Exec
+        Command="git clean -xdf obj"
+        WorkingDirectory="$(MSBuildThisFileDirectory)"
+    />
```

Comment:
Could this use a regular `<RemoveDir/>`?

Reply:
I updated this to use removdir.